components/Tile/Tile.tsx

```diff
@@ -0,0 +1,109 @@
+import styled from "styled-components";
+import css from "@styled-system/css";
+import Box from "components/Box";
+import Flex from "components/Flex";
+import Icon, { IconName } from "components/Icon";
+import Link from "components/Link";
+
+import bgHref from "./assets/bg.svg";
+
+export interface TileSetProps {
+  children: React.ReactNode;
```

Comment:
```suggestion
  children: ReactElement<typeof Tile> | Array<ReactElement<typeof Tile>>;
```
It's can be useful if we specify a little bit stricter types here.